lib/fluent/plugin/parser_groonga_log.rb

```diff
@@ -34,6 +34,7 @@ def configure(conf)
       def parse(text)
         @parser.parse(text) do |statistic|
           event_time = Fluent::EventTime.from_time(statistic.timestamp)
+          statistic.timestamp = statistic.timestamp.to_s
```

Comment:
Don't change `statistic` object directly. It's not a good manner.

Reply:
Use `Time#iso8601` to ensure using standard format.

Reply:
@kou I've fixed and pushed above feedback.